sdk/lib/rtl/amd64/dynfntbl.c

```diff
@@ -154,6 +154,10 @@ RtlAddFunctionTable(
                                            FunctionTable[i].EndAddress);
     }
 
+    /* Adjust the margins to be absolute addresses */
+    dynamicTable->MinimumAddress += BaseAddress;
+    dynamicTable->MaximumAddress += BaseAddress;
```

Comment:
There's no need to check for any kind of weird rollover, in case someone is trying to run a malicious crafted x64 exe, or manually invoke the function with intentionally wrong parameters?

Reply:
if someone is running a malicious crafted x64 exe, why would they bother to break exception handling and not just run whatever they want to run?